### PR TITLE
Form validation

### PR DESCRIPTION
**Pull request by Stanleeeeee**
This form validation doesn't accept uppercases in the email. it was made possible using 
- Javascript

### CODE REVIEW

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @Stanleeeeee & @felixodette 👋,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

# Status: Invalid Code Review Request



**_You have submitted a project, that has failed linters. Kindly fix linters errors._**

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by Stanleeeeee — COMMENTED** [review]
This is the form validation using Javascript. All linters errors have been corrected

File: index.js [hunk 1]
```diff
@@ -42,3 +42,208 @@ function resizeWindow() {
...
+const email = document.getElementsById('email');
+const lowerCaseRegex = /[A-Z]/;
+
+form.addEventListener('submit', (e) => {
+  if (lowerCaseRegex.test(email.value.trim())) {
+    e.preventDefault();
+    const errorMessage = document.querySelector('small');
+    errorMessage.classList.remove('hidden');
+  }
+});
```
Comment (codepantha) [inline index.js]: - [ ] Your form submits even when the user enters an uppercase email. The correct behaviour should be an error that tells the user to make sure the email is lowercase only. Please fix this 🙏

**Review by codepantha — CHANGES_REQUESTED** [review]
Hi @Stanleeeeee 👋,

### STATUS: REQUIRED CHANGES ♻️  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


### To Highlight 💡 
- Meaningful commit messages ✅ 
- Professional README ✅ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Stanleeeeee — COMMENTED** [review]
Form validation using javascript. I made corrections requested by the reviewer

File: index.js [hunk 2]
```diff
@@ -42,3 +42,206 @@ function resizeWindow() {
...
+/* Form validation */
+const form = document.getElementById('contact-form');
+const email = document.getElementById('email');
+const lowerCaseRegex = /^[a-z][a-z0-9\-_.]+@[a-z]{2,}\.[a-z0-9]{2,}$/;
+
+form.addEventListener('submit', (e) => {
+  if (lowerCaseRegex.test(email.value.trim())) {
+    e.preventDefault();
+    const errorMessage = document.querySelector('small');
+    errorMessage.classList.remove('hidden');
```
Comment (AnselemOdims) [inline index.js]: - [ ] Good job on making it work but please remove the HTML validation and make sure it only uses the Javascript validation with the error message appearing close to the submit button

**Review by AnselemOdims — CHANGES_REQUESTED** [review]
### Hi @Stanleeeeee , 
Good job implementing some of the changes requested by the previous reviewer.

## Required Changes ♻️
- [ ] Please also check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity! You can tag me using @AnselemOdims for a quicker response

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by Stanleeeeee — COMMENTED** [review]
Made corrections to the email validation
- It uses lowercasing only for validation
-  It throws an error if you do otherwise

**Review by smunozmo — APPROVED** [review]
Hi @Stanleeeeee, 👋 

Your project is complete!  There is nothing else to say other than... it's time to merge it! 🚀 
Very nice work, congratulations! 🎉 

### APPROVED! 👏 👏 👏 




Cheers and Happy coding! 🤓 

✅  Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**